invenio_accounts/testutils.py

```diff
@@ -35,6 +35,7 @@
 
 import flask
 import flask_login
+from faker import Faker
```

Comment:
IMHO it's not cool if we have dependency in package which is installed only for tests without checking it with `pkg_resources` at least.